metadeploy/api/models.py

```diff
@@ -918,6 +905,7 @@ def fail(self, error):
     def complete(self, config):
         self.status = ScratchOrgJob.Status.complete
         self.config = config
+        self.org_id = config["org_id"]
```

Comment:
@wlonk This clearly isn't strictly necessary, but should make the `ORG_CHANGED` event in `notify_org_result_changed` work seamlessly for both persistent orgs and scratch orgs.